src/django_bootstrap5/html.py

```diff
@@ -17,8 +21,24 @@ def render_link_tag(url):
     return render_tag("link", attrs=attrs, close=False)
 
 
+def has_prefix(name, prefixes):
+    """Return whether the name has one of the given prefixes."""
+    return any(name.startswith(f"{prefix}_") for prefix in prefixes)
+
+
+def hyphenate(attr_name):
+    """Return the hyphenated version of the attribute name."""
+    return attr_name.replace("_", "-")
+
+
 def render_tag(tag, attrs=None, content=None, close=True):
     """Render an HTML tag."""
+    prefixes = get_bootstrap_setting("hyphenate_attribute_prefixes") or []
```

Comment:
Not specifying should result in the default value being used.
```suggestion
    prefixes = get_bootstrap_setting("hyphenate_attribute_prefixes") or ["data"]
```